packages/atomic/src/components/atomic-result/atomic-result-cell-desktop.pcss

```diff
@@ -34,14 +34,18 @@
       @mixin set-font-size var(--atomic-text-sm);
 
       margin-bottom: 1rem;
-      height: 2rem;
+      --line-height: 2rem;
+      line-height: var(--line-height);
+      max-height: calc(2 * var(--line-height) + 0.5rem);
     }
 
     atomic-result-section-actions {
       @mixin set-font-size var(--atomic-text-base);
 
       margin-top: 2.25rem;
-      height: 2.5rem;
+      --line-height: 2.5rem;
+      line-height: var(--line-height);
+      max-height: calc(2 * var(--line-height) + 0.5rem);
```

Comment:
This actions section appears on the same row as the badges section, it should've had the same height in the first place.